[DOCS-11525] Update Cloud SIEM landing page (#30784)
DataDog/documentation: content/en/security/cloud_siem/_index.md

---

{{< learning-center-callout header="Join an enablement webinar session" hide_image="true" btn_title="Sign Up" btn_url="https://www.datadoghq.com/technical-enablement/sessions/?tags.topics-0=Security">}}
  Learn how Datadog Cloud SIEM and Cloud Security elevate your organization's threat detection and investigation for dynamic, cloud-scale environments.
{{< /learning-center-callout >}}

## Overview

Datadog Cloud SIEM (Security Information and Event Management) is a security data analysis and correlation system. It enables your entire security operations team to view, detect, investigate, and respond to security issues. Leveraging Datadog's scalable platform, Cloud SIEM ingests telemetry from both cloud and on‑premises systems using the Datadog Agent and API-based integrations.

Effective security response requires speed, context, insight, and automation. By continuously analyzing incoming data, Cloud SIEM detects threats, generates actionable security signals, and correlates those signals across multiple sources. This empowers your team to investigate incidents and respond quickly.

> To keep your team on top of the latest attacks, Datadog also has a team of threat researchers who analyze petabytes of telemetry across cloud and on-premises systems to identify emerging threats and attacker behaviors. See Datadog Security Labs to read articles about their recent investigations.

Author: updated

### Risk insights

{{< img src="security/security_monitoring/landing/03_risk_insights.png" alt="" style="width:100%;" >}}

Review the risky entities in your environment.

- *Top Risky Entities:* Shows which entities have the highest risk scores at this time, and includes risk score, entity name, entity type, last risk score update timestamp, and risk score change. Click on an entity to view more details and take action.
- *Entity Type Breakdown:* Shows which types of entities are most common in your environment. Click to view a list of entities filtered by that type.
- *Entities risk score breakdown:* View entities by severity. Click to view the entities explorer with a filtered view of entities by severity.

### Threat map

{{< img src="security/security_monitoring/landing/04_threat_map.png" alt="" style="width:100%;" >}}

Get insights from where the signals in your environment are getting generated.

- *Top IPs by Country Distribution:* See which IPs are generating the most signals with a breakdown of important and less important signals. Also, use the map to see a list of signals by country.

> - *Signals by Country:* See the proportional breakdown from where signals are being generated. Click on the wedges to see signals by country and state/province, and spot signals being generated from unexpected countries.

Author: ```suggestion
- *Signals by Country:* See the proportional breakdown from where signals are being generated. Click on a wedge on the pie chart to see signals by country and state/province, and spot signals being generated from unexpected countries.
```

> - *Signals by ISP Provider:* Review from which ISPs the signals are coming. Drill in by provider and location.

Author: "Drill in" is a phrase that we try to avoid: https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/2752086538/Words+and+Phrases+to+Avoid#%E2%80%9Cdrill-down%E2%80%9D

```suggestion
- *Signals by ISP Provider:* Review from which ISPs the signals are coming. Click on a wedge on the pie chart to scope down by provider and location.
```

### Security overview

{{< img src="security/security_monitoring/landing/05_security_overview.png" alt="" style="width:100%;" >}}

A high-level overview of all signals.

> - *Signal Distribution:* On the left side of the page are a collection of KPIs showing signals by severity and trend over the selected time window.

Author: ```suggestion
- *Signal Distribution:* On the left side of the page, see signals grouped by severity and trend over the selected time window.
```

> On the right is a graph allowing users to see a break down of signal activity by severity, source, and resolution. Click on any area to drill into a filtered view of the signal explorer.

Author: ```suggestion
On the right side, see a break down of signal activity by severity, source, and resolution. Click on a node in the sankey diagram to see signals in the Signal Explorer filtered to the specifics of that node.
```

> - *Mean Time to Respond to Signals:* Displays KPIs showing how quickly your team responds. Sets a signal to `under review` or `archive` by signal severity.

Author: ```suggestion
- *Mean Time to Respond to Signals:* See KPIs of how quickly your team responds. Sets a signal to `under review` or `archive` by signal severity.
```

### MITRE ATT&CK coverage

{{< img src="security/security_monitoring/landing/06_mitre_coverage.png" alt="" style="width:100%;" >}}

Detection rule coverage and signal activity by MITRE ATT&CK tactics and techniques.

- *Techniques with at least 1 rule:* See how many techniques are covered by the detection rules currently enabled in your environment.

> - *Rule Density KPIs*: See how many techniques have high, medium, or low density or no rules at all. Click to view a filtered MITRE map.

Author: ```suggestion
- *Rule density tiles*: See how many techniques have high, medium, or low density or no rules at all. Click on a tile to view a filtered MITRE map.
```

> - *Signals per Tactic View:* See which MITRE ATT&CK tactics are generating signals. Click to view the signal explorer filtered by tactic. Use the toggle to show which tactics have the most rules mapped to it. When viewing by rule, clicking will create a filtered detection rule explorer view by that tactic.

Author: ```suggestion
- *Signals per Tactic View:* See which MITRE ATT&CK tactics are generating signals. Click a wedge on the pie chart to view the Signal Explorer filtered by tactic. Click the dropdown and select **Rules count** to see which tactics have the most rules mapped to it. When viewing by rule count, clicking a wedge on the pie chart creates a detection rule explorer view filtered by that tactic.
```

> - *Signals per Technique View:* See which MITRE ATT&CK techniques are generating signals. Click to view the signal explorer filtered by technique. Use the toggle to show which techniques have the most rules mapped to it. When viewing by rule, clicking will create a filtered detection rule explorer view by that technique.

Author: ```suggestion
- *Signals per Technique View:* See which MITRE ATT&CK techniques are generating signals. Click on a pie chart wedge to view the signal explorer filtered by technique. Click the dropdown and select **Rules count** to see which techniques have the most rules mapped to it. When viewing by rule count, clicking on a pie chart wedge creates a detection rule explorer view filtered by that technique.
```

### Detection rules performance

{{< img src="security/security_monitoring/landing/07_detection_rule_performance.png" alt="" style="width:100%;" >}}

Gain a deeper understanding of detection rule performance. This section works best if you triage signals in Cloud SIEM.

> - *MTTD KPIs for Cloud SIEM:* See the Mean Time to Detect (MTTD) across all signals. The KPIs below show MTTD for critical, high, and medium signals.

Author: ```suggestion
- *MTTD KPIs for Cloud SIEM:* See the Mean Time to Detect (MTTD) across all signals. The tiles below show MTTD for critical, high, and medium signals.
```

> - *Signal Activity:* View signal trends over the selected time window. Click the filters at the bottom to drill in by severity, which can be useful for identifying unusual spikes or drops.

Author: ```suggestion
- *Signal Activity:* View signal trends over the selected time window. Select the severity checkboxes at the bottom of the bar graph to scope by severity, which can be useful for identifying unusual spikes or drops.
```

> - *Rules by Important Signal Change (1 week):* Shows which rules have increased important signal activity compared to the week prior. Clicking takes you to a filtered view of the Signal Explorer by that rule name.

Author: ```suggestion
- *Rules by Important Signal Change (1 week):* See which rules have increased important signal activity compared to the week prior. Click a rule name to view signals in the Signal Explorer filtered by that rule name.
```

> - *Signals by severity change (1 week):* View how the severities across all signals have changed compared to the week prior. Click to view a filtered view of the signal explorer by that severity.

Author: ```suggestion
- *Signals by severity change (1 week):* View how the severities across all signals have changed compared to the week prior. Click on a severity to view signals with that severity in the Signal Explorer.
```

> - *Important Signals by Archived Reason:* See how many signals were archived by each of the archive reasons. Click to view the Signal Explorer filtered by that archive reason.

Author: ```suggestion
- *Important Signals by Archived Reason:* See how many signals were archived by each archive reason. Click on a reason to view the Signal Explorer filtered by that archive reason.
```

> - *Rules Archived with True Positive (Malicious):* See which rules were archived as `True Positive: Malicious`. Click to view the signals in the signal explorer.

Author: ```suggestion
- *Rules Archived with True Positive (Malicious):* See which rules were archived as `True Positive: Malicious`. Click a rule to view the signals in the Signal Explorer.
```

> - *Rules Archived with True Positive (Benign):* See which rules were archived as `True Positive: Benign`. Click to view the signals in the signal explorer.

Author: ```suggestion
- *Rules Archived with True Positive (Benign):* See which rules were archived as `True Positive: Benign`. Click a rule to view the signals in the Signal Explorer.
```

> - *Rules by False Positive Rate:* Shows which rules are the noisiest by calculating the percentage of signals that are marked as false positive out of all the signals generated by a rule. Click to view a filtered view of the signal explorer by that rule.

Author: ```suggestion
- *Rules by False Positive Rate:* See which rules are the noisiest by calculating the percentage of signals that are marked as false positive out of all the signals generated by a rule. Click on a rule to view signals for that rule in the Signal Explorer.
```

1. Navigate to [Cloud SIEM][3].
1. Click **Enable Cloud SIEM**.
1. Follow the onboarding steps.

See the [Getting Started Guide][4] for more detailed setup instructions.

## Key features

### Security and observability

Cloud SIEM embeds both cloud and on-premises telemetry directly into security workflows to accelerate investigation and response. And with a shared platform that brings DevOps and Security teams together, organizations can break down silos and respond to threats collaboratively and efficiently.

### Flexible cost control

As your organization scales, controlling the ingestion cost of security logs without compromising visibility is critical. Cloud SIEM is integrated with Datadog Log Management so you can choose the retention and querying capability of your security logs to balance cost efficiency and your threat detection needs. The available options are:
- [Standard indexing][6] for logs that need to be queried frequently and retained short term.
- [Flex Logs][7] for logs that need to be retained long-term, but sometimes need to be queried urgently.
- [Log Archives][8] for logs that are infrequently queried and need to be stored long-term, such as audit logs.

> ## Cloud SIEM Overview Page

Author: ```suggestion
## Cloud SIEM Overview page
```

Navigate to the [Cloud SIEM Overview dashboard][2]. Use this dashboard to see key security insights and act on common workflows for security analysts, security and detection engineers, and Security Operations Center (SOC) managers. From the Overview page, you can:
- Access important signals, open cases, and high-risk entities.
- Complete onboarding tasks and review content‑pack health.
- View and investigate top signals by geography or ISP.
- Analyze signals and rules by MITRE ATT&CK tactics.
- Track detection performance (MTTD, false‑positive rates).
- Read the latest [Security Labs][1] research and release notes.

Click **Customize Page** to reorder or hide modules so you can see what is important to you.

> Learn more below about each section of the Overview page.

Author: ```suggestion
Learn more about each Cloud SIEM Overview page section below.
```

### Security coverage

{{< img src="security/security_monitoring/landing/01_security_coverage.png" alt="Security coverage sections showing 11 active and 1 broken content packs and a bar graph of logs analyzed by Cloud SIEM" style="width:100%;" >}}

Remain aware of any data processing issues or coverage gaps.

#### Enabled content packs and integrations

View enabled content packs and integrations across the critical categories for providing comprehensive security coverage. Hover over each section of the horizontal bar to see which content packs are enabled in each category.

#### Content pack and logs health KPIs

> See whether any content packs or integrations are in warning or broken states so that you can quickly resolve any coverage gaps. Click a status tile to view the affected content packs.

Author: ```suggestion
See whether any content packs or integrations are in warning or broken states so that you can resolve any coverage gaps. Click a status tile to view the affected content packs.
```

- [Standard indexing][6] for logs that need to be queried frequently with the most compute.
- [Flex Logs][7] for logs that need to be retained long-term, but sometimes need to be queried urgently.
- [Log Archives][8] for logs that are infrequently queried and need to be stored long-term.

### Guided security data onboarding

Cloud SIEM [Content Packs][9] are a curated set of Datadog integrations specifically useful for security teams. Each content pack has instructions on how to configure the integration and what is included, such as detection rules, out-of-the-box interactive dashboards, parsers, and SOAR workflows. Content Packs highlight actionable insights specific to that integration to help you investigate security issues.

### Content pack health monitoring

After a content pack is activated, the content pack gives you the integration's health status and provides troubleshooting steps if something goes wrong so you can get back up and operational as fast as possible.

### Log search and analysis

> Build searches in the Log Explorer using facets or by clicking fields directly in the logs. Or use Bits AI and natural language search to find important security events. With built-in group-by and table lookup functions as well as pattern analysis and visualizations, security teams can get security insights from their data.  See [Log Search Syntax][10] for more information.

Author: @jnhunsberger  Should we link to the Log Explorer doc as well? https://docs.datadoghq.com/logs/explorer/

Reviewer: Yes

### Content pack health monitoring

After a content pack is activated, the content pack gives you the integration's health status and provides troubleshooting steps if something goes wrong so you can get back up and operational as fast as possible.

### Log search and analysis

Build searches in the Log Explorer using facets or by clicking fields directly in the logs. Or use Bits AI and natural language search to find important security events. With built-in group-by and table lookup functions as well as pattern analysis and visualizations, security teams can get security insights from their data.  See [Log Search Syntax][10] for more information.

## Get started

If you don't already have a Datadog account, sign up for a [free trial][2]. After you log in to your Datadog account:

1. Navigate to [Cloud SIEM][3].
1. Click **Enable Cloud SIEM**.
1. Follow the onboarding steps.

See the [Getting Started Guide][4] for more detailed setup instructions.

## Cloud SIEM Overview page

> Navigate to the [Cloud SIEM Overview dashboard][3]. Use this dashboard to see key security insights and act on common workflows for security analysts, security and detection engineers, and Security Operations Center (SOC) managers. From the Overview page, you can:

Reviewer: The overview page is not a dashboard per se. I recommend changing the two references to `dashboard` to `page`.

- Access important signals, open cases, and high-risk entities.
- Complete onboarding tasks and review content‑pack health.
- View and investigate top signals by geography or ISP.
- Analyze signals and rules by MITRE ATT&CK tactics.

> - Track detection performance (MTTD, false‑positive rates).

Reviewer: should we spell out MTTD (mean time to detect)?

#### Top IPs by country distribution

See which IPs are generating the most signals with a breakdown of important and less important signals. Also, use the map to see a list of signals by country.

#### Signals by country

See the proportional breakdown from where signals are being generated. Click on a pie chart wedge to see signals by country and state/province, and spot signals being generated from unexpected countries.

#### Signals by ISP provider

Review from which ISPs the signals are coming. Click on a pie chart wedge to scope down by provider and location.

### Security overview

{{< img src="security/security_monitoring/landing/05_security_overview.png" alt="" style="width:100%;" >}}

A high-level overview of all signals.

#### Signal Distribution

On the left side of the section, see signals grouped by severity and trend over the selected time window.
On the right side, see a break down of signal activity by severity, source, and resolution. Click on a node in the sankey diagram to see signals in the Signal Explorer filtered to the specifics of that node.

#### Mean Time to Respond to Signals

> See KPIs of how quickly your team responds. Sets a signal to `under review` or `archive` by signal severity.

Author: > Sets a signal to `under review` or `archive` by signal severity.

@jnhunsberger i'm kind of confused by this sentence...who or what sets the signal to `under review` or `archive`?

Reviewer: Good catch, May. Let's change it to:
"See KPIs of how quickly your team responds to signals. Click to view the signals that were set to `under review` or `archive` filtered by severity.

Author: ```suggestion
See KPIs of how quickly your team responds. Click a severity tile to view signals set to `under review` or `archive` and filtered to the selected severity.
```

#### Signal activity

View signal trends over the selected time window. Select the severity checkboxes at the bottom of the bar graph to scope by severity, which can be useful for identifying unusual spikes or drops.

#### Rules by important signal change (1 week)

See which rules have increased important signal activity compared to the week prior. Click a rule name to view signals in the Signal Explorer filtered by that rule name.

#### Signals by severity change (1 week)

View how the severities across all signals have changed compared to the week prior. Click on a severity to view signals with that severity in the Signal Explorer.

#### Important signals by archived reason

See how many signals were archived by archive reason. Click on a reason to view the Signal Explorer filtered by that archive reason.

#### Rules archived with true positive (malicious)

See which rules were archived as `True Positive: Malicious`. Click on a rule to view the signals in the Signal Explorer.

#### Rules archived with true positive (benign)

See which rules were archived as `True Positive: Benign`. Click on a rule to view the signals in the Signal Explorer.

#### Rules by false positive rate

> See which rules are the noisiest by calculating the percentage of signals that are marked as false positive out of all the signals generated by a rule. Click on a rule to view signals for that rule in the Signal Explorer.

Reviewer: Can we please add in the Diver Deeper section from the Google Doc here?

---

{{< learning-center-callout header="Join an enablement webinar session" hide_image="true" btn_title="Sign Up" btn_url="https://www.datadoghq.com/technical-enablement/sessions/?tags.topics-0=Security">}}
  Learn how Datadog Cloud SIEM and Cloud Security elevate your organization's threat detection and investigation for dynamic, cloud-scale environments.
{{< /learning-center-callout >}}

## Overview

> Datadog Cloud SIEM (Security Information and Event Management) is a security data analysis and correlation system. It enables your entire security operations team to view, detect, investigate, and respond to security issues. Leveraging Datadog's scalable platform, Cloud SIEM ingests telemetry from both cloud and on‑premises systems using the Datadog Agent and API-based integrations.

Reviewer 2: ```suggestion
Datadog Cloud SIEM (Security Information and Event Management) is a security data analysis and correlation system. It enables your entire security operations team to view, detect, investigate, and respond to security issues. Leveraging Datadog's scalable platform, Cloud SIEM ingests telemetry from both cloud and on‑premises systems through the Datadog Agent and API-based integrations.
```

> Effective security response requires speed, context, insight, and automation. By continuously analyzing incoming data, Cloud SIEM detects threats, generates actionable security signals, and correlates those signals across multiple sources. This empowers your team to investigate incidents and respond quickly.

Reviewer 2: ```suggestion
Effective security response requires speed, context, insight, and automation. Cloud SIEM continuously analyzes incoming data to detect threats, generate actionable security signals, and correlate them across multiple sources. This empowers your team to investigate incidents and respond quickly.
```

To keep your team on top of the latest attacks, Datadog also has a team of threat researchers who analyze petabytes of telemetry across cloud and on-premises systems to identify emerging threats and attacker behaviors. See [Datadog Security Labs][1] to read articles about their recent investigations.

### Security and observability

> Cloud SIEM embeds both cloud and on-premises telemetry directly into security workflows to accelerate investigation and response. And with a shared platform that brings DevOps and Security teams together, organizations can break down silos and respond to threats collaboratively and efficiently.

Reviewer 2: This seems to repeat the Overview section. Do you need both sections?

Author: It is a bit redundant. I think it would make sense to remove. @jnhunsberger thoughts?

Reviewer: I would like to keep this copy. It makes specific points about cloud/on-prem and bringing DevOps and Security together which are not directly covered in the Overview section.

### Flexible cost control for security data

As your organization scales, controlling the ingestion cost of security logs without compromising visibility is critical. Cloud SIEM is integrated with Datadog Log Management so you can choose the retention and querying capability of your security logs to balance cost efficiency and your threat detection needs. The available options are:
- [Standard indexing][6] for logs that need to be queried frequently with the most compute.
- [Flex Logs][7] for logs that need to be retained long-term, but sometimes need to be queried urgently.
- [Log Archives][8] for logs that are infrequently queried and need to be stored long-term.

### Guided security data onboarding

> Cloud SIEM [Content Packs][9] are a curated set of Datadog integrations specifically useful for security teams. Each content pack has instructions on how to configure the integration and what is included, such as detection rules, out-of-the-box interactive dashboards, parsers, and SOAR workflows. Content Packs highlight actionable insights specific to that integration to help you investigate security issues.

Reviewer 2: ```suggestion
Cloud SIEM [Content Packs][9] are a curated set of Datadog integrations designed for security teams. Each content pack has instructions on how to configure the integration and what is included, such as detection rules, out-of-the-box interactive dashboards, parsers, and SOAR workflows. Content Packs highlight actionable insights specific to each integration to help you investigate security issues.
```

### Content pack health monitoring

> After a content pack is activated, the content pack gives you the integration's health status and provides troubleshooting steps if something goes wrong so you can get back up and operational as fast as possible.

Reviewer 2: ```suggestion
After a content pack is activated, it gives you the integration's health status and provides troubleshooting steps if something goes wrong so you can get back up and operational as fast as possible.
```

### Log search and analysis

Build searches in the Log Explorer using facets or by clicking fields directly in the logs. Or use Bits AI and natural language search to find important security events. With built-in group-by and table lookup functions as well as pattern analysis and visualizations, security teams can get security insights from their data. See [Log Explorer][11] and [Log Search Syntax][10] for more information.

## Get started

If you don't already have a Datadog account, sign up for a [free trial][2]. After you log in to your Datadog account:

1. Navigate to [Cloud SIEM][3].
1. Click **Enable Cloud SIEM**.
1. Follow the onboarding steps.

See the [Getting Started Guide][4] for more detailed setup instructions.

> ## Cloud SIEM Overview page

Reviewer: Let's save that for a separate effort. I would like to get this out as is.

Navigate to the [Cloud SIEM Overview page][3]. Use this page to see key security insights and act on common workflows for security analysts, security and detection engineers, and Security Operations Center (SOC) managers. From the Overview page, you can:
- Access important signals, open cases, and high-risk entities.
- Complete onboarding tasks and review content‑pack health.

> - View and investigate top signals by geography or ISP.

Author: Good catch

```suggestion
- View and investigate top signals by geography or internet service provider (ISP).
```

#### Logs analyzed

View logging trends across your top log sources and identify any unusual spikes or drops. Click on the legend at the bottom to explore trends on a per source basis.

### Important signals and cases

{{< img src="security/security_monitoring/landing/02_important_signals_cases.png" alt="" style="width:100%;" >}}

See important events happening in your environment, such as:

#### Recent open signals grouped by rule

See signals grouped by rule name and sorted by severity to get an overview of the most important signals in your environment. Click on a signal or a severity pill to see more details in a filtered view in the Signal Explorer.

#### Recent open security cases

Use [Case Management][5] for signals of interest that require additional analysis. View the current security cases in your environment and click on a case to see more details.

### Risk insights

{{< img src="security/security_monitoring/landing/03_risk_insights.png" alt="" style="width:100%;" >}}

Review the risky entities in your environment.

#### Top risky entities

> See which entities have the highest risk scores at this time. Click on an entity to view more details and take action.

Reviewer 2: ```suggestion
See the entities with the highest risk scores. Click an entity to view more details and take action.
```

#### Entity type breakdown

> See which types of entities are most common in your environment. Click on a pie chart wedge to view a list of entities filtered by that type.

Reviewer 2: ```suggestion
View the most common entity types in your environment. Click a pie chart wedge to filter the list of entities by type.
```

#### Entities risk score breakdown

View entities by severity. Click a severity tile to see a list of entities with that severity.

### Threat map

{{< img src="security/security_monitoring/landing/04_threat_map.png" alt="" style="width:100%;" >}}

Get insights from where the signals in your environment are getting generated.

#### Top IPs by country distribution

See which IPs are generating the most signals with a breakdown of important and less important signals. Also, use the map to see a list of signals by country.

#### Signals by country

See the proportional breakdown from where signals are being generated. Click on a pie chart wedge to see signals by country and state/province, and spot signals being generated from unexpected countries.

#### Signals by ISP provider

> Review from which ISPs the signals are coming. Click on a pie chart wedge to scope down by provider and location.

Reviewer 2: ```suggestion
Review which ISPs are sending signals. Click on a pie chart wedge to scope down by provider and location.
```
